File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/AuthorityLimitConfiguration.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class AuthorityLimitConfiguration
    {
        public int AuthorityLimitConfigurationId { get; set; } // AuthorityLimitConfigurationId (Primary key)
        public AuthorityLimitConfigurationTypeEnum AuthorityLimitConfigurationType { get; set; } // AuthorityLimitConfigurationTypeId
        public AuthorityLimitValueTypeEnum AuthorityLimitValueType { get; set; } // AuthorityLimitValueTypeId
        public AuthorityLimitItemTypeEnum AuthorityLimitItemType { get; set; } // AuthorityLimitItemTypeId
        public int Value { get; set; } // Value
        public string PermissionName { get; set; } // PermissionName (length: 50)
        public bool IsDeleted { get; set; } // IsDeleted
        public string ModifiedBy { get; set; } // ModifiedBy (length: 50)
        public System.DateTime ModifiedDate { get; set; } // ModifiedDate
        public string CreatedBy { get; set; } // CreatedBy (length: 50)
        public System.DateTime CreatedDate { get; set; } // CreatedDate

    }
}

File: Web/RMA-Web-Shell/projects/shared-models-lib/src/lib/authority-limits/authority-limit-configuration.ts
import { AuthorityLimitConfigurationTypeEnum } from "../enums/authority-limits/authority-limit-configuration-type-enum";
import { AuthorityLimitItemTypeEnum } from "../enums/authority-limits/authority-limit-item-type-enum";
import { AuthorityLimitValueTypeEnum } from "../enums/authority-limits/authority-limit-value-type-enum";

export class AuthorityLimitConfiguration {
    authorityLimitConfigurationId: number;
    authorityLimitConfigurationType: AuthorityLimitConfigurationTypeEnum;
    authorityLimitValueType: AuthorityLimitValueTypeEnum;
    authorityLimitItemType: AuthorityLimitItemTypeEnum;
    value: number;
    permissionName: string;

    isDeleted: boolean;
    modifiedBy: string;
    modifiedDate: Date;
    CreatedBy: string;
    CreatedDate: Date;
}